terraform/projects/pr-dashboard/rds.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_db_subnet_group" "rds" {       
+  name = "default-db"                        
+  description = "default subnets"            
+  subnet_ids = ["${aws_subnet_wpt_id}"]
```

Comment:
You can use our existing subnets with `${module.subnet.cidr_blocks}`

That is available because the subnet module outputs it here:
https://github.com/bocoup/infrastructure-web-platform/blob/master/terraform/modules/aws/subnet/main.tf#L10

...and is consumed by this project here:
https://github.com/bocoup/infrastructure-web-platform/blob/master/terraform/projects/pr-dashboard/network.tf#L34-L40